main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+)
+
+// Converts a JSON file containing people and ages to a human-readable txt file
+func main() {
+
+	type Person struct {
+		FirstName string
+		Age       int
+	}
+
+	var people []Person
+
+	data, err := ioutil.ReadFile("./testdata/people.json")
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+
+	err = json.Unmarshal(data, &people)
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+
+	for i := range people {
+		fmt.Println(people[i].FirstName + " is " + strconv.Itoa(people[i].Age) + " years old.")
```

Comment:
Whenever we want to interpolate data of arbitrary type into a string, we can reach for `fmt.Sprintf`! Then it's easy to construct a string that looks like our desired output, only with placeholders where the data should go:

```suggestion
		return fmt.Sprintf("%s is %d years old.", p.FirstName, p.Age)
```